docs/best-practices/continuous-integration/codemagic.md

```diff
@@ -0,0 +1,28 @@
+# Codemagic Integration
+
+[Codemagic](https://codemagic.io/) is a cloud-based CI/CD tool for Flutter apps that you can use for continuous delivery together with *fastlane*. Codemagic has *fastlane* pre-installed, so you can easily run your *fastlane* scripts as part of the Codemagic build process. 
+
+When using *fastlane* for beta distribution to third-party services, you will need to add the API keys / secrets to Codemagic as secure [environment variables](https://docs.codemagic.io/building/environment-variables/). For example, in order to publish to Crashlytics, you need to add the following environment variables:

```

Comment:
Crashlytics was shut down after Google bought them and incorporated them into Firebase. Maybe we can give another example of beta distribution service, or update to mention Firebase?

Reply:
Replaced with Firebase App Distribution example